retrofit-adapters/rxjava2/src/main/java/retrofit2/adapter/rxjava2/RxJava2CallAdapterFactory.java

```diff
@@ -59,7 +59,15 @@
    * by default.
    */
   public static RxJava2CallAdapterFactory create() {
-    return new RxJava2CallAdapterFactory(null);
+    return new RxJava2CallAdapterFactory(null, false);
+  }
+
+  /**
+   * Returns an instance which creates asynchronous observables. Applying
```

Comment:
Ah, right, "network requests will be performed by `Call` dispatcher (ie OkHttp) instead of blocking a scheduler or application thread"? 

I mean, would be great to include some pros of using it into javadoc (and release notes)!